tests/loops/test_flow_warnings.py

```diff
@@ -1,45 +0,0 @@
-# Copyright The PyTorch Lightning team.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-import warnings
-
-from pytorch_lightning import Trainer
-from tests.helpers.boring_model import BoringModel
-
-
-class TestModel(BoringModel):
-    def training_step(self, batch, batch_idx):
-        acc = self.step(batch[0])
-        return acc
-
-
-def test_no_depre_without_epoch_end(tmpdir):
```

Comment:
This was a check for a warning in https://github.com/PyTorchLightning/pytorch-lightning/pull/3849 but it doesn't exist anymore